agent/agent_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package agent
+package main_test
```

Comment:
Package names ending in "_test" would be an exception to the rule; it's an old idiom that is mentioned (briefly) in the official Go docs (e.g. [here](https://tip.golang.org/cmd/go/#hdr-Test_packages) ).

Additionally, the reasoning behind that guideline is to keep import names short, but no one is going to be importing a "\<something\>_test" package, so that concern isn't really applicable.